File: bench/bench.js
// JS: loop, string ops, arithmetic
let s = 0;
for (let i = 1; i <= 10000; i++) {
    s += i;
}
console.log(`Sum: ${s}`);

// String building
let result = "";
for (let i = 1; i <= 100; i++) {
    result = result + "x";
}
console.log(`Len: ${result.length}`);

// Array ops
let arr = [];
for (let i = 1; i <= 1000; i++) {
    arr.push(i);
}
console.log(`Array len: ${arr.length}`);

File: bench/bench.py
# Python: loop, string ops, arithmetic
s = 0
for i in range(1, 10001):
    s += i
print(f"Sum: {s}")

# String building
result = ""
for i in range(1, 101):
    result = result + "x"
print(f"Len: {len(result)}")

# Array ops
arr = []
for i in range(1, 1001):
    arr.append(i)
print(f"Array len: {len(arr)}")
